src/collections/_documentation/performance/distributed-tracing.md

```diff
@@ -201,6 +201,20 @@ You can pass many different options to tracing, but it comes with reasonable def
 - If available: Browser Resources (Scripts, CSS, Images ...)
 - If available: Browser Performance API Marks
 
+*tracingOrigins Option*
+ 
+The default value of `tracingOrigins` is : `new ApmIntegrations.Tracing({tracingOrigins: ['localhost', /^\//]})`. This options will attach the `sentry-trace` header to all outgoing XHR/fetch requests that contain a string in the list or match the regex. In case your frontend is making requests to a different domain you might want to add it there so we attach the `sentry-trace`.
+
+*Example:*
+
+- Your frontend is running `mywebapp.com`
```

Comment:
```suggestion
- A frontend application is served from `example.com`
```

`example.com` is a reserved domain for documentation, I'd be cautious using other domains we don't own or control.